doc/users/next_whats_new/mathtext.rst

```diff
@@ -0,0 +1,11 @@
+``matplotlib.mathtext`` now supports *overset* and *underset* LaTeX symbols
+---------------------------------------------------------------------------
+
+`.mathtext` now supports *overset* and *underset*, called as 
+``\overset{body}{annotation}`` or ``\underset{body}{annotation}``, where 
```

Comment:
OK, but now this is wrong again, right?

Reply:
Yeah, I blindly copied the suggestion, let me fix that